templates/vue/src/components/modals/NodeModal/forms/ContentForm/ActivityForm.vue

```diff
@@ -140,6 +140,55 @@
                 Audio recorder
               </b-form-checkbox>
             </b-form-group>
+            <b-form-group>
+              <b-form-checkbox
+                v-model="question.answerTypes.list.enabled"
+                data-qa="question-answer-list"
+                switch
+              >
+                List Answer
+              </b-form-checkbox>
+              <div v-if="question.answerTypes.list.enabled" class="mt-2 pl-4 ml-2">
+                <div class="list-options">
+                  <div class="checkbox-input">
+                    <div class="list-checkbox">
+                      <label for="min-field">Min</label>
+                    </div>
+                    <div class="list-input">
+                      <b-form-input
+                        id="min-field"
+                        v-model="question.answerTypes.list.minFields"
+                        placeholder="Min # fields"
```

Comment:
If the form input type is number, how come the placeholder is a string?

Reply:
The placeholder is just text to prompt the user to know what to put into the input field when it's empty